frameworks/PHP/symfony/.env

```diff
@@ -0,0 +1,17 @@
+# This file is a "template" of which env vars need to be defined for your application
+# Copy this file to .env file for development, create environment variables when deploying to production
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
```

Comment:
this one should not be commited, so should be removed here
it would also ensure that it is not used actually (since this is for dev ony)

for the console, the values should be defined in frameworks/PHP/symfony/setup.sh I think instead.